README.md

```diff
@@ -55,6 +55,7 @@ If you are the maintainer of a project, you can add our badge by copying the cod
 
 - **[Algo Repo](https://github.com/orbitturner/reposTDAlgoP3Simplon)** - Repository of some Algorithmic exercice for those who want to make their first steps into the Dev World. (*NOT MAINTAINED ANYMORE*) **by [@orbitturner](https://github.com/orbitturner)**
 - **[Amazon Textract CSV](https://github.com/JoeDalton24/amazon-textract-response-tocsv)** - Amazon Textract Response to CSV export tables and extract key-value pairs in form documents from JSON returned by Amazon Textract into a comma-separated values (CSV) file. **by [@JoeDalton24](https://github.com/JoeDalton24)**
+- **[ANGULAR VPS DEPLOYMENT](https://github.com/orbitturner/NG-TINY-CONF-2022-DEMO-APP)** - Ma présentation durant le *NgTinyConf2022* consistait a montrer comment déployer une application Angular dans un Serveur Privé Virtuel en passant par la sécurité, la management et la configuration de NGINX ainsi que quelques bonus.. **by [@orbitturner](https://github.com/orbitturner)**
```

Comment:
Hello @orbitturner est-ce que tu peux traduire la description en ENG pour harmoniser le tout.

Reply:
Okay je m'en charge thanks for the review 🤗🤗🤗